packages/Pill/src/Pill.styles.js

```diff
@@ -0,0 +1,76 @@
+import { css } from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+const pillColorStyles = {
+  "#d7d7d7": css`
+    color: ${tokens.color.black};
+
+    svg {
+      fill: ${tokens.color.black};
+    }
+  `,
+
+  noRisk: css`
+    background: ${tokens.color.blackLighten60};
+    color: ${tokens.color.black};
+  `,
+
+  lowRisk: css`
+    background: #299a7a;
+  `,
+
+  mediumRisk: css`
+    background: #c9af28;
+  `,
+
+  highRisk: css`
+    background: #cd3c44;
+  `,
+};
+
+const pillSizeStyles = {
+  small: css`
+    border-radius: ${tokens.space} / 50%;
+    font-weight: bold;
+    line-height: ${stylers.spacer(2)};
```

Comment:
Would a pill ever need to break its text into two lines?

Reply:
This makes the height of the pill

Reply:
Maybe just `height` would make more sense then?